test/fixtures/listing-valid.json

```diff
@@ -28,14 +28,14 @@
   "unitsTotal": 1,
   "price": {
     "currency": "ETH",
-    "amount": "200"
+    "amount": "0.033"
   },
   "commission": {
     "currency": "OGN",
-    "amount": "10"
+    "amount": "0"
```

Comment:
Why are you changing the commission to 0?

Reply:
Does it make sense to explicitly test creating a listing with commission and a listing without?

Reply:
@cuongdo Just changing the amount in the fixture. Origin-js API actually does not support making an offer with commission yet.

@micahalcorn Yes it does. I started making the test and then realized that our API doesn't support this yet.

I have already started working on supporting commissions in the API, just wanted to keep those changes in separate PRs.

Reply:
@tyleryasaka the listing commission work is being done in a different branch. We hadn't gotten to offers yet.

Reply:
@tyleryasaka https://github.com/OriginProtocol/origin-js/pull/465

Reply:
Ok cool. Well then I will add tests for the offer price checks, and will have to leave these fixtures with a commission of `0` for now since we don't have commission yet. Once the tests are in we can merge this.